py/private/py_pytest_main.bzl

```diff
@@ -88,6 +88,6 @@ def py_pytest_main(name, py_library = default_py_library, deps = [], data = [],
         srcs = [test_main],
         tags = tags,
         visibility = visibility,
-        deps = deps,
+        deps = deps + ["@aspect_rules_py//py/private/pytest_shard"],
```

Comment:
this is technically incorrect in a macro, where a string will turn into a label in the caller's context and they might not use this name for the apparent repo.

Instead you should `Label("//py/private/pytest_shard")` so you explicitly turn into a label in the macro's context where it will be "the repository I'm in"

Reply:
TIL! Thanks. I originally just had `//py/private/pytest_shard` which didn't obviously work when used from an external repository.

Reply:
also TIL